CMakeLists.txt

```diff
@@ -22,6 +22,41 @@ add_executable(
         src/scene.h
         src/scene.cpp
         src/render_call_info.h
+        ${CMAKE_CURRENT_BINARY_DIR}/include/shader_path.hpp
+        ${rgen_shader_path}
```

Comment:
Why is the `rgen_shader_path` included in the executable?

Reply:
I forget to delete this line.
In CMake function ```compile_glsl_help```, I will add shader source code to executable so that I could edit these shaders on Visual Studio.